Localization/pl-PL.hjson

```diff
@@ -2,154 +2,154 @@ Mods: {
-				// LeftClickToChangeRange: Left Click to [c/{0}:change range]
-				// LeftClickToToggle: Left Click to [c/{0}:toggle on/off]
-				// RightClickToShowCurrentRange: Right Click to [c/{0}:show current range]
-				// RightClickToTurnOff: Right Click to [c/{0}:turn off]
-				// MagnetIsOff: Magnet is off
-				// MagnetOff: Magnet off
-				// Off: off
-				// Range: range: {0}
-				// NextRange: | next: {0}
+				DisplayName: Magnes na przedmioty
+				CurrentRange: "{$Buffs.ItemMagnetBuff.CurrentRange}"
+				CurrentVelocity: "{$Buffs.ItemMagnetBuff.CurrentVelocity}"
+				CurrentAcceleration: "{$Buffs.ItemMagnetBuff.CurrentAcceleration}"
+				LeftClickToChangeRange: Naciśnij lewy przycisk myszy aby [c/{0}:zmienić zasięg]
+				LeftClickToToggle: Naciśnij lewy przycisk myszy aby [c/{0}:włączyć/wyłączyć]
+				RightClickToShowCurrentRange: Naciśnij prawy przycisk myszy aby [c/{0}:pokazać aktualny zasięg]
+				RightClickToTurnOff: Naciśnij prawy przycisk myszy aby [c/{0}:wyłączyć]
+				MagnetIsOff: Magnes jest wyłączony
+				MagnetOff: Magnes wyłączony
+				Off: wyłączony
+				Range: zasięg: {0}
+				NextRange: | następny: {0}
 			}
 		}
 
 		Configs: {
 			Config: {
-				// DisplayName: Config
+				DisplayName: Ustawienia konfiguracji
 
 				Headers: {
-					// PresetItemWhitelist: Preset Item Whitelist
-					// CustomItemFilter: Custom Item Filter
-					// General: General
-					// MagnetBehavior: Magnet Behavior (Only works ingame)
-					// ResultingMagnetStats: Resulting Magnet Stats
+					PresetItemWhitelist: Wbudowane filtry przedmiotów
+					CustomItemFilter: Własne filtry przedmiotów
+					General: Ogólne
+					MagnetBehavior: Zachowanie Magnesu (Działa tylko w grze)
+					ResultingMagnetStats: Aktualne właściwości magnesu
 				}
 
 				Hearts: {
-					// Label: "[i:58] Hearts"
-					// Tooltip: Toggle if hearts should be picked up by the magnet
+					Label: "[i:58] Serca"
+					Tooltip: Przełącz, jeśli magnes ma przyciągać serca.
 				}
 
 				ManaStars: {
-					// Label: "[i:184] Mana Stars"
-					// Tooltip: Toggle if mana stars should be picked up by the magnet
+					Label: "[i:184] Gwiazdy Many"
+					Tooltip: Przełącz, jeśli magnes ma przyciągać gwiazdy many.
 				}
 
 				Coins: {
-					// Label: "[i:73] Coins"
-					// Tooltip: Toggle if coins should be picked up by the magnet
+					Label: "[i:73] Monety"
+					Tooltip: Przełącz, jeśli magnes ma przyciągać monety.
 				}
 
 				PickupEffect: {
-					// Label: "[i:FragmentNebula] Pickup Effect Items"
-					// Tooltip: Toggle if items like nebula armor boosters or modded items like the music notes from Thorium should be picked up by the magnet
-				}
+					Label: "[i:FragmentNebula] Przedmioty z efektem przy podnoszeniu"
+					Tooltip: Przełącz, jeśli magnes ma przyciągać rzeczy takie jak wzmocnienia pochodzące ze Zbroi Nebuli, czy też rzeczy pochodzące z modyfikacji takie jak nuty z Thorium.
+                                }
```

Comment:
whitespace change. should be tabs, not spaces

Reply:
Fixed